addons/account/views/account.xml

```diff
@@ -31,6 +31,7 @@
 
     <template id="assets_frontend" name="account assets" inherit_id="web.assets_frontend">
         <xpath expr="." position="inside">
+            <link rel="stylesheet"  href="/account/static/src/scss/section_and_note_backend.scss"/>
```

Comment:
Maybe this could go in `assets_common` as it was already in `assets_backend`

Reply:
So far this section/note style is specific to account (sale and invoice), I guess we'll move it into common if another module needs it at some point.

Reply:
But what I mean is that want a different style in the backend and frontend. We like how sections are displayed in the backend currently, but they  don't look good on the frontend and we plan to change it.